docs/root/intro/version_history.rst

```diff
@@ -25,6 +25,7 @@ Version history
 * config: use Envoy cpuset size to set the default number or worker threads if :option:`--cpuset-threads` is enabled.
 * config: added support for :ref:`initial_fetch_timeout <envoy_api_field_core.ConfigSource.initial_fetch_timeout>`. The timeout is disabled by default.
 * cors: added :ref:`filter_enabled & shadow_enabled RuntimeFractionalPercent flags <cors-runtime>` to filter.
+* dubbo_proxy: support dubbo proxy filter.
```

Comment:
can you merge master and bump this to 1.11?

Reply:
done